internal/keyspan/test_utils.go

```diff
@@ -5,13 +5,13 @@
 package keyspan
 
 import (
+	"bytes"
 	"fmt"
 	"go/token"
 	"io"
 	"reflect"
 	"strconv"
 	"strings"
-	"testing"
```

Comment:
while this doesn't depend `testing`, it does still depend on `datadriven` on the line below this, so moving it from _test.go to a linked file meant we picked up datadriven's init-time CLI flag registrations

Reply:
#3326